mappings/net/minecraft/client/util/math/Vector3f.mapping

```diff
@@ -30,6 +32,11 @@ CLASS net/minecraft/class_1160 net/minecraft/client/util/math/Vector3f
 		ARG 4 z
 	METHOD method_23846 add (Lnet/minecraft/class_1160;)V
 		ARG 1 vector
+	METHOD method_23847 multiplyComponentwise (Lnet/minecraft/class_1160;F)V
+		ARG 1 vector
+		ARG 2 ratio
+	METHOD method_23848 modify (Lit/unimi/dsi/fastutil/floats/Float2FloatFunction;)V
```

Comment:
`transform`

Reply:
I don't know. This one's a bit tricky, because it modifies this vector in place using the supplied function.

Reply:
Let's use `map`, like `Stream.map`/`Optional.map`

Reply:
```suggestion
	METHOD method_23848 map (Lit/unimi/dsi/fastutil/floats/Float2FloatFunction;)V
```

Reply:
Do not use `map` for in place modifications. `Stream.map` and `Optional.map` are not in place operations.

Reply:
So guess `modify` is fine.